src/cittadini/01_core.tf

```diff
@@ -50,11 +50,11 @@ module "apim_api_cittadini_v1" {
   service_url  = "https://${var.dns_zone_internal_entry}.${var.apim_dns_zone_prefix}.${var.external_domain}/pagopaarcbe/arc"
 
   content_format = "openapi"
-  content_value = templatefile("./api/cittadini/v1/_openapi.json.tpl", {
-    host = local.apim_hostname
+  content_value = templatefile("./api/cittadini/v1/area_riservata_cittadino_openapi.yml", {
+    hostname = local.apim_hostname
   })
 
-  xml_content = templatefile("./api/cittadini/v1/_base_policy.xml", {
-    hostname = local.cittadini_ingress_be
+  xml_content = templatefile("./api/cittadini/v1/area_riservata_cittadino_base_policy.xml", {
+    cittadini_ingress_hostname = local.cittadini_ingress_hostname
```

Comment:
is this field needed? The base policy doesn't contain any variable to be injected, am i missing something?

Reply:
the field cittadini_ingress_hostname is a typo, I used it initially in the policy